### ISSUE TITLE
Expand list of binaries

### ISSUE THREAD

**Issue by Clay-Mysten** [MystenLabs/sui#1733]
In https://github.com/MystenLabs/sui/pull/1611 we discovered this list is incomplete:
https://docs.sui.io/build/install#binaries

Francois has filed the related:
[[doc, UX] split our binaries from the Sui crate by use case](https://github.com/MystenLabs/sui/issues/1639#)
#1639 

I'm now filing this issue to update the install docs for our forthcoming event as we will be using the install docs extensively.

I'm asking for Patrick's help and Francois' review and guidance.

**Comment by Clay-Mysten** [comment]
This looks like the latest list:

```
   Compiling sui-adapter v0.1.0 (/Users/claymurphy/.cargo/git/checkouts/sui-6ac459c53b1b685a/217cad9/sui_programmability/adapter)
   Compiling sui_core v0.1.0 (/Users/claymurphy/.cargo/git/checkouts/sui-6ac459c53b1b685a/217cad9/sui_core)
    Finished release [optimized] target(s) in 3m 56s
  Installing /Users/claymurphy/.cargo/bin/bench
  Installing /Users/claymurphy/.cargo/bin/bench_configure
  Installing /Users/claymurphy/.cargo/bin/key_tool
  Installing /Users/claymurphy/.cargo/bin/remote_load_generator
  Installing /Users/claymurphy/.cargo/bin/rpc-server
  Installing /Users/claymurphy/.cargo/bin/sui
  Installing /Users/claymurphy/.cargo/bin/sui-move
  Installing /Users/claymurphy/.cargo/bin/validator
  Installing /Users/claymurphy/.cargo/bin/wallet
   Installed package `sui v0.1.0 (https://github.com/MystenLabs/sui.git?branch=devnet#217cad94)` (executables `bench`, `bench_configure`, `key_tool`, `remote_load_generator`, `rpc-server`, `sui`, `sui-move`, `validator`, `wallet`)
claymurphy@Clays-MacBook-Pro bin % 
``

It would be great if folks in the know wanted to augment the list here:
https://github.com/MystenLabs/sui/blob/main/doc/src//build/install.md#binaries

Adding Sam, Todd, and Janet for reassignment.

**Comment by patrickkuo** [comment]
We will need https://github.com/MystenLabs/sui/issues/1639 to clean up the binaries, we are currently building all binaries in the `sui/sui` package, we need to group them by use case and only keep the ones relevant to doc user before we can update the doc.

**Comment by Clay-Mysten** [comment]
Now that we are planning to start publishing from the `devnet` branch and staging from `main`, can we add all of these?

**Comment by gdanezis** [comment]
hey @Clay-Mysten is this done now?

**Comment by Clay-Mysten** [comment]
Sadly, no. The list has been ever evolving. Are we done for now, @patrickkuo ? If so, I will pick this back up.

**Comment by Clay-Mysten** [comment]
Hi @patrickkuo and team, I've started this update in:
https://github.com/MystenLabs/sui/pull/2711

It would be good to link to docs for the other binaries if/where they exist. I know we have some benchmark docs under `narwhal` for instance:
https://github.com/MystenLabs/narwhal/tree/main/benchmark

Just not sure they are relevant here. Please advise.

**Comment by Clay-Mysten** [comment]
Thanks to @asonnino for his review of PR #2711. @patrickkuo , I will work with you to update this list once again after `sui` and `wallet` have been merged. Thanks, all!

**Comment by Clay-Mysten** [comment]
@patrickkuo , this is now complete, correct?:

```
sui - The Sui CLI tool contains subcommands for enabling genesis of validators and accounts, starting the Sui network, and [building and testing Move packages](https://docs.sui.io/build/move), as well as a [client](https://docs.sui.io/build/cli-client) for interacting with the Sui network.
 [rpc-server ](https://docs.sui.io/build/json-rpc)- run a local Sui gateway service accessible via an RPC interface.
```

When will we list the other binaries I added here?
https://github.com/MystenLabs/sui/pull/2711

**Comment by Clay-Mysten** [comment]
OK, thanks.

All, please reopen or file anew if/when we should advertise all of the other binaries we have available.

**Comment by Clay-Mysten** [comment]
Adding @damirka , who thankfully caught these issues in Discord:
https://mysten-labs.slack.com/archives/C034XH78HNU/p1657844453319569
https://discord.com/channels/916379725201563759/925108748551323649/997277113864429598

Reopening to take care of some discrepancies in our docs, where we still use `sui-move` instead of `sui move`. See the attached file:
[sui-move-refs.txt](https://github.com/MystenLabs/sui/files/9116830/sui-move-refs.txt)

Damir, please feel free to include the input you saw on our tutorials being out of date too. I can always move it from here to another go/sui-doc-bug. Thanks!

**Comment by Clay-Mysten** [comment]
Just noting @jaredcosulich is a savior, contributing to these latest updates in:
https://github.com/MystenLabs/sui/pull/3216

Thank you, Jared!

**Comment by Clay-Mysten** [comment]
+Xun

Here should be the last two PRs to fix the `sui-move` > `sui move` changes:
https://github.com/MystenLabs/sui/pull/3219
https://github.com/MystenLabs/sui/pull/3220

Asking @patrickkuo and @lxfind to review them, respectively. Thanks, all.

**Comment by patrickkuo** [comment]
> +Xun
> 
> Here should be the last two PRs to fix the `sui-move` > `sui move` changes: #3219 #3220
> 
> Asking @patrickkuo and @lxfind to review them, respectively. Thanks, all.

Thanks! I thought I have changed all of them when I made the code changes 😞 , it's hard to catch 'em all
